powersimdata/network/constants/region/division.py

```diff
@@ -0,0 +1,65 @@
+class DivisionMapping:
+    """State/Country mapping for grid models.
+
+    :param pandas.DataFrame zone: information on zones of a grid model.
+    """
+
+    def __init__(self, zone):
+        self.abv = set(zone["abv"])
+        self.abv2loadzone = {
+            k: set(v)
+            for k, v in zone.groupby("abv")["zone_name"].unique().to_dict().items()
+        }
+        self.abv2id = {k: set(v) for k, v in zone.groupby("abv").groups.items()}
+        self.id2abv = {k: v for k, v in zone["abv"].to_dict().items()}
+        self.abv2interconnect = dict(zip(zone["abv"], zone["interconnect"]))
+
+
+class USADivisionMapping(DivisionMapping):
+    """State mapping for USA grid models
+
+    :param pandas.DataFrame zone: information on zones of a grid model.
+    """
+
+    def __init__(self, zone):
+        super().__init__(zone)
+        self.state = set(zone["state"])
+        self.state_abbr = set(zone["abv"])
+        self.state2loadzone = {
+            k: set(v)
+            for k, v in zone.groupby("state")["zone_name"].unique().to_dict().items()
+        }
```

Comment:
Maybe we could do `self.state2loadzone = zone.groupby("state")["zone_name"].apply(set).to_dict()` instead. Same comment below.

Reply:
Found other places where it can be applied

Reply:
Done